WalletWasabi/TorSocks5/TorSocks5Client.cs

```diff
@@ -40,14 +40,7 @@ public bool IsConnected
 			{
 				try
 				{
-					if (TcpClient == null || !TcpClient.Connected || TcpClient.GetStream() == null)
-					{
-						return false;
-					}
-					else
-					{
-						return true;
-					}
+					return TcpClient != null && TcpClient.Connected;
```

Comment:
Why?

Reply:
Because GetStream never returns a null, it is a redundant check.